workflow/workflow-demos-test/src_test/test/bpm/TestOrder.java

```diff
@@ -0,0 +1,72 @@
+package test.bpm;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmElement;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmProcess;
+import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
+import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.ivy.notification.impl.NotificationRepository;
+import ch.ivyteam.ivy.notification.query.NotificationQuery;
+import ch.ivyteam.ivy.scripting.objects.Tuple;
+import ch.ivyteam.ivy.workflow.CaseState;
+import workflow.order.Order;
+import workflow.order.Person;
+
+@IvyProcessTest
+public class TestOrder
+{
+  private static final BpmProcess ORDER_PROCESS = BpmProcess.name("OrderProcess");
+  
+  private static final BpmElement START_ORDER = ORDER_PROCESS.elementName("startOrder");
+  private static final BpmElement INPUT_ORDER = ORDER_PROCESS.elementName("Input Order");
+  private static final BpmElement PROCESS_ORDER = ORDER_PROCESS.elementName("Process Order");
+  private static final BpmElement DELIVER_ORDER = ORDER_PROCESS.elementName("Deliver Order");
+  private static final BpmElement SEND_INVOICE = ORDER_PROCESS.elementName("Send Invoice");
+  private static final BpmElement BOOK_PAYMENT = ORDER_PROCESS.elementName("Book Payment");
+
+  private static final Person CUSTOMER = new Person();
+  private static final Order ORDER = new Order();
+
+  static {
+    CUSTOMER.setName("Doe");
+    CUSTOMER.setSurname("John");
+    ORDER.setProduct("Ferrari Testarossa");
+  }
+  
+  @Test
+  void runProcess(BpmClient bpmClient)
+  {
```

Comment:
legacy fromatting ;-)

Reply:
Can change in a a different PR. But then all demos.

Reply:
Have changed this class to legazy formatting only. Will convert every demo to new formatting in a new PR after my holidays.